restdocs-api-spec-jsonschema/src/main/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGenerator.kt

```diff
@@ -146,6 +148,9 @@ class JsonSchemaFromFieldDescriptorsGenerator {
                     .build()
             )
         } else {
+            if (propertyField?.fieldDescriptor?.let { isRequired(it) } == true) {
```

Comment:
This `== true` looks weird, but IntelliJ recommends this as a solution (problem is that `isRequired` requires a non-null type and returns a non-null Boolean, but the entire expression could evaluate to null already if the optional parameter is null).

Reply:
Yes that's a shorthand I use a lot in my daily Kotlin code to deal with nullable boolean. It's a shorthand for
```kotlin
if(nullableBoolean ?: false)
```
I can implement it as `?: false` or keep it the way it is, as you want